gravitee-apim-repository/gravitee-apim-repository-mongodb/src/main/java/io/gravitee/repository/mongodb/management/upgrade/upgrader/index/events/PropertiesApiIdUpdatedAtIndexUpgrader.java

```diff
@@ -22,16 +22,17 @@
 /**
  * @author GraviteeSource Team
  */
-@Component("EventsLatestPropertyApiIdIndexUpgrader")
-public class PropertyApiIdIndexUpgrader extends IndexUpgrader {
+@Component("EventsPropertiesApiIdUpdatedAtIndexUpgrader")
+public class PropertiesApiIdUpdatedAtIndexUpgrader extends IndexUpgrader {
 
     @Override
     protected Index buildIndex() {
         return Index
             .builder()
-            .collection("events_latest")
```

Comment:
Do we have other indices for the "events_latest" collection? Because this collection is highly used for GW synchronization process

Reply:
https://github.com/gravitee-io/gravitee-api-management/tree/4.0.x/gravitee-apim-repository/gravitee-apim-repository-mongodb/src/main/java/io/gravitee/repository/mongodb/management/upgrade/upgrader/index/eventslatest